src/app/app-routing.module.ts

```diff
@@ -1,22 +1,19 @@
 import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
 import {RouterModule, Routes} from '@angular/router';
-import {PostsComponent} from './posts/posts.component';
-import {PostDetailsComponent} from './post-details/post-details.component';
-import {NotFoundComponentComponent} from './not-found-component/not-found-component.component';
-import {AboutUsComponent} from 'src/app/about-us/about-us.component';
-import {ContactComponent} from 'src/app/contact/contact.component';
-import {PageComponent} from 'src/app/page/page.component';
+import {NotFoundComponentComponent} from './core/components/not-found-component/not-found-component.component';
+import {PageComponent} from './core/components/page/page.component';
+import {PostsComponent} from './features/posts/posts.component';
+import {PostDetailsComponent} from './features/posts/post-details/post-details.component';
+import {ContactComponent} from './features/contact/contact.component';
 
 const routes: Routes = [
   {path: '', component: PostsComponent},
-  /*  {path: 'posts/:id', component: PostDetailsComponent},*/
   {
     path: 'posts', children: [
       {path: ':id', component: PostDetailsComponent}
     ]
   },
```

Comment:
ok for now, but /post reladtedroute paths should be kept in child routing module (as mentioned in v6.1)

Reply:
I corrected.  /post related route paths kept in child routing module